src/NserviceBus.Transport.Msmq/InstanceMapping/InstanceMappingFileFeature.cs

```diff
@@ -25,7 +27,7 @@ protected override void Setup(FeatureConfigurationContext context)
             }
 
             var checkInterval = context.Settings.Get<TimeSpan>(CheckIntervalSettingsKey);
-            var endpointInstances = context.Routing.EndpointInstances;
+            var endpointInstances = context.Settings.Get<EndpointInstances>();
```

Comment:
@Particular/nservicebus-maintainers Do we want to consider making `Routing` public? I was able to work around not having it.